tests/trailing-space.cpp

```diff
@@ -1,10 +1,10 @@
 // This comment includes trailing space. 
- 
+
 /*
  * This multi-line comment	
  * includes trailing space.	
  */
-  
+
```

Comment:
Trailing spaces on this otherwise blank line have been removed.  Was that intentional?

Reply:
Confirmed as intentional. This is the reverse test case from blank-trailing-space: Trailing spaces on an otherwise blank line have been intentionally eliminated. Trailing spaces on an otherwise non-blank line have been preserved, since these are the explicit focus of the test case.